packages/core/lib/src/models/media.dart

```diff
@@ -0,0 +1,43 @@
+/// Enum for the type of media in a tweet
+enum MediaType {
+  /// The media is a video
+  video,
+
+  /// The media is a GIF
+  gif,
+
+  /// The media is a single image
+  photo,
+}
+
+/// Tweet Media object
+class Media {
```

Comment:
The media object in the Twitter API is more complex than this. But I thought to reduce complexity to save time with implementation but if you want me to replicate the exact API data structure I can do so.

Reply:
Works for me